test/server/test_data/server/watchdogs_bootstrap_with_deprecated_field.yaml

```diff
@@ -1,7 +0,0 @@
-watchdogs:
-  main_thread_watchdog:
```

Comment:
I guess this config file was never used?

Reply:
Yep, it was added in an intermediate commit, when we were splitting watchdog into multiple watchdogs. But it didn't get used in the end, and I missed removing it (until now).